packages/polling-controller/src/PollingController.ts

```diff
@@ -0,0 +1,150 @@
+import { BaseControllerV2 } from '@metamask/base-controller';
+import type {
+  RestrictedControllerMessenger,
+  StateMetadata,
+} from '@metamask/base-controller';
+import type { NetworkClientId } from '@metamask/network-controller';
+import type { Json } from '@metamask/utils';
+import { v1 as random } from 'uuid';
```

Comment:
Hmm, why not use v4? It's my understanding that v1 is not quite as random, and that v4 is the better choice in most situations